csharp-basic-topics/FileAccessModifierInCsharp/FileAccessModifierInCsharp/ContentUploadService.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Content;
+
+static file class Logger
```

Comment:
And should we apply file to both of them? Can we do it only for one of them?

Reply:
Can be only one. In the draft I mentioned that to fix the error we need to add the modifier to at least one class.